frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -52,43 +57,80 @@ function renderReports(reports) {
       </Tag>
     )) : '';
 
+    const history = useHistory();
     const idKey = `my_alerts_${id}`;
     const idLink = `/activity-reports/${id}`;
+    const contextMenuLabel = `View activity report ${id}`;
     const statusClassName = `smart-hub--table-tag-status smart-hub--status-${status}`;
 
+    const { isOpen, openModal, closeModal } = useModal();
+    const onDelete = (reportId) => {
+      // Once backend deletion method is implemented, this fake
+      // if statement is no longer needed, just deleteReport(...)
+      // eslint-disable-next-line no-constant-condition
+      if (false) deleteReport(reportId);
+      fetchReports();
+      closeModal();
+    };
+    const ConnectModal = connectModal(DeleteReportModal);
+
+    const menuItems = [
+      {
+        label: 'Open report',
```

Comment:
The table below labels this as `View`, will want to update them to match (can probably also drop `report` on both labels)